tests/cli/test_cmds_status.py

```diff
@@ -23,17 +23,25 @@
 
 from paasta_tools import utils
 from paasta_tools.cli.cmds import status
+from paasta_tools.cli.cmds.status import apply_args_filters
 from paasta_tools.cli.cmds.status import missing_deployments_message
-from paasta_tools.cli.cmds.status import paasta_args_mixer
 from paasta_tools.cli.cmds.status import paasta_status
 from paasta_tools.cli.cmds.status import report_invalid_whitelist_values
-from paasta_tools.cli.cmds.status import report_status
 from paasta_tools.cli.cmds.status import verify_instances
 from paasta_tools.cli.utils import NoSuchService
 from paasta_tools.cli.utils import PaastaCheckMessages
 from paasta_tools.cli.utils import PaastaColors
 
 
+def make_fake_instance_conf(cluster, service, instance, deploy_group=None):
+    conf = MagicMock()
+    conf.get_cluster = lambda: cluster
```

Comment:
Out of curiosity, is this just another way to write 'conf.get_cluster.return_value = cluster'?
Or is there any difference?